packages/mui-material/src/Chip/Chip.js

```diff
@@ -337,6 +344,7 @@ const Chip = React.forwardRef(function Chip(inProps, ref) {
     onDelete,
     onKeyDown,
     onKeyUp,
+    rounded,
```

Comment:
```suggestion
    rounded = false,
```

Reply:
just passing by, why do you need a `false` default value? (I think you don't)